src/utils/index.js

```diff
@@ -3,7 +3,7 @@ import html2canvas from 'html2canvas';
 import * as jsPDF from 'jspdf';
 
 const move = (array, element, delta) => {
-  const index = array.indexOf(element);
```

Comment:
indexOf is good for primitive types; in this case, we are dealing with a complex object so accessing by id is more reasonable